pkg/dispatchcli/cmd/create_policy.go

```diff
@@ -0,0 +1,97 @@
+///////////////////////////////////////////////////////////////////////
+// Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+///////////////////////////////////////////////////////////////////////
+
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/vmware/dispatch/pkg/dispatchcli/i18n"
+
+	policy "github.com/vmware/dispatch/pkg/identity-manager/gen/client/policy"
+	models "github.com/vmware/dispatch/pkg/identity-manager/gen/models"
+)
+
+var (
+	createPolicyLong = i18n.T(`Create a dispatch identity manager policy`)
+
+	createPolicyExample = i18n.T(`  dispatch create policy test_policy -s xyz@example.com -a get,update -r image,api,function`)
```

Comment:
Whitespace in the beginning

Reply:
Also, it will be nice to some proper examples like with long flags
```
# Create a policy by specifying a single subject, action, and resource
dispatch create policy example_policy --subject user1@example.com --action get --resource *

# Create a policy by specifying multiple subjects, actions, and resources
dispatch create policy example_policy --subject user1@example.com,user2@example.com --action get,create,delete,update --resource image,function,base-image,secret

````

Reply:
An issue here is that when specify * as value, cmd will try to list all file in current directory. '*' has to be put inside quote marks. That's saying `--r "*"` instead of `--r *`. Is there a workaround?